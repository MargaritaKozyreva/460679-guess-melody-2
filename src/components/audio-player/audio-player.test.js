import React from "React";
import AudioPlayer from "./audio-player.jsx";
import renderer from "react-test-renderer";

describe(`AudioPlayer initial`, () => {

  const onPlayButtonClick = jest.fn();

  const mockProps = {
    isPlaying: true,
    onPlayButtonClick,
    src: `https://upload.wikimedia.org/wikipedia/track.ogg`
  };

  it(`renders correctly`, () => {
    const tree = renderer.create(<AudioPlayer {...mockProps}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
