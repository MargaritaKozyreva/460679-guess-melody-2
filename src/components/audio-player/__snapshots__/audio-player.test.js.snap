// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer initial renders correctly 1`] = `
Array [
  <button
    className="track__button track__button--pause"
    disabled={false}
    onClick={[Function]}
    type="button"
  />,
  <div
    className="track__status"
  >
    <audio
      src="https://upload.wikimedia.org/wikipedia/track.ogg"
    />
  </div>,
]
`;
